src/App.js

```diff
@@ -35,29 +37,40 @@ const store = createStore(
 );
 sagaMiddleware.run(actions);
 
+const generateClassName = createGenerateClassName();
+const jss = create(jssPreset());
+// We define a custom insertion point that JSS will look for injecting the styles in the DOM.
+jss.options.insertionPoint = document.getElementById('jss-insertion-point');
+
 const withDefault = component => () => (
   <Default>{React.createElement(component)}</Default>
 );
 
 const App = () => (
-  <ErrorBoundary>
-    <Provider store={store}>
-      <ConnectedRouter history={history}>
-        <Switch>
-          <Route exact path="/" component={withDefault(withRouter(Home))} />
-          {Object.keys(routes).map(route => (
+  <JssProvider jss={jss} generateClassName={generateClassName}>
+    <ErrorBoundary>
+      <Provider store={store}>
+        <ConnectedRouter history={history}>
+          <Switch>
             <Route
-              path={route}
-              component={withDefault(withRouter(routes[route]))}
-              key={route}
+              exact
```

Comment:
In order to replicate the behaviour of Drupal right now we would need to check "access content overview" on this route. Of course we can do this later, but I think access checking on routes, not just endpoints, is something we should keep in mind somehow.